docs/src/ice_dynamics.md

```diff
@@ -69,9 +75,9 @@ end
 to_graphviz(glens_law)
 ```
 
-# Composing models
+## Composing models
 
-We can use operadic composition to specify how our models come together. In this example, we have two Decapodes, and two quantities that are shared between them.
```

Comment:
This needs to go back

Reply:
In that case, `operadic composition` should be better explained or at least have a link to a resource describing it. This term is not a common one nor do we define it here ourselves.

Reply:
Ok, link to Sophie's paper

Reply:
This one? https://arxiv.org/abs/2105.12282